apps/wallet-mobile/src/features/Swap/useCases/StartSwapScreen/CreateOrder/EditBuyAmount/SelectBuyTokenFromListScreen/SelectBuyTokenFromListScreen.tsx

```diff
@@ -144,21 +93,22 @@ const TokenList = () => {
       )}
 
       <FlashList
-        data={filteredTransformedList}
-        renderItem={({item: tokenForList}: {item: TokenForList}) => (
+        data={filteredTokenList}
+        renderItem={({item: token}: {item: Balance.Token}) => (
           <Boundary loading={{fallback: <AmountItemPlaceholder style={styles.item} />}}>
             <SelectableToken
-              tokenForList={tokenForList}
-              disabled={tokenForList.id !== wallet.primaryTokenInfo.id}
+              token={token}
+              disabled={token.info.id !== wallet.primaryTokenInfo.id}
```

Comment:
This prop is defined as a type but not used in the component. I'm not sure if it's really necessary.

Reply:
This component was forked from the Send feature, some bits may be leftovers that don't work here. Like this disabled.